server/app/models/fusor/deployment.rb

```diff
@@ -16,7 +16,7 @@ class Deployment < ActiveRecord::Base
     # on update because we don't want to validate the empty object when
     # it is first created
     validates_with Fusor::Validators::DeploymentValidator, on: :update
-    validates_associated :openstack_deployment
+    validates_associated :openstack_deployment, on: :update
```

Comment:
On initial create, the openstack_deployment is not valid so it would fail to create.  Updates (save) is being called with validate: false.  openstack_deployment now matches the behavior of the deployment parent object.